### PR TITLE
Added lighting section and lighting topics to API

### PR DESCRIPTION
**Pull request by roni-kreinin**

### CODE REVIEW

File: docs_versioned_docs/version-ros2humble/ros/api/lighting.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+toc_max_heading_level: 5
+---
+
+:::note
+
+This section only applies to platforms with corner lights (D1X0, W200, R100).
+
+:::
+
+Corner lighting is unified for platforms that support it. Lighting is controlled by the [lighting node](https://github.com/clearpathrobotics/clearpath_common/tree/humble/clearpath_platform/src/lighting) which defines multiple lighting states.
```
Comment (tonybaltovski) [inline docs_versioned_docs/version-ros2humble/ros/api/lighting.mdx]: Let's add which platforms indeed do support it here.  Also, should we state that not all platforms will show  the states presented.  Such as W200 will not show shore power states?

**Review by tonybaltovski — CHANGES_REQUESTED** [review]
One minor nit but looks great! Especially the gifs!

File: docs_versioned_docs/version-ros2humble/ros/api/lighting.mdx (hunk 1 above)
Comment (roni-kreinin) [reply docs_versioned_docs/version-ros2humble/ros/api/lighting.mdx]: I mentioned the supported platforms in the note at the top, but I could move that to this paragraph. And I'll add another column to the table indicating which platforms support the state.

Comment (tonybaltovski) [reply docs_versioned_docs/version-ros2humble/ros/api/lighting.mdx]: The column should be enough.

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**